From 7ecb36442d12f197310a336a0e963badd238f35e Mon Sep 17 00:00:00 2001
From: Kenneth Hsu <kennethhsu@gmail.com>
Date: Sun, 23 Apr 2017 10:37:02 -0700
Subject: [PATCH 37/37] staging: rtl8723bs: declare private function as static

This fixes a sparse warning regarding an undeclared symbol.  Since the
function is private to rtw_recv.c, it should be declared as static.

Signed-off-by: Kenneth Hsu <kennethhsu@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/staging/rtl8723bs/core/rtw_recv.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/drivers/staging/rtl8723bs/core/rtw_recv.c
+++ b/drivers/staging/rtl8723bs/core/rtw_recv.c
@@ -1717,7 +1717,8 @@ sint wlanhdr_to_ethhdr(union recv_frame
 }
 
 /* perform defrag */
-union recv_frame *recvframe_defrag(struct adapter *adapter, struct __queue *defrag_q)
+static union recv_frame *recvframe_defrag(struct adapter *adapter,
+					  struct __queue *defrag_q)
 {
 	struct list_head	 *plist, *phead;
 	u8 *data, wlanhdr_offset;
